added event 20230207 (#123)
AzugBe/azugbe.github.io: _posts/events/2023-02-07---connected-manufacturing-using-azure-services.md

---
layout: single
title: "2023-02-07 - Connected manufacturing using azure services"

> date: 2022-12-07 18:00:00 +0000

Reviewer: ```suggestion
date: 2023-02-07 18:00:00 +0000
```

comments: true
published: true
categories: ["events"]
tags: ["Events"]
author: Kris van der Mast
---

> In our first session of the year we're going to start in Ghent.

Reviewer: ```suggestion
In our first session of the year, we'll have a look at DataOps and what it can mean for you. See you in Ghent?
```

## Agenda

> ### Connected manufacturing using azure services

Reviewer: ```suggestion
### Connected manufacturing using Azure services
```
